installation.rst

```diff
@@ -67,6 +67,48 @@ If installed via ``pip``, Conan version can be updated with:
 The upgrade shouldn't affect the installed packages or cache information. If the cache becomes inconsistent somehow, you may want to remove its content by deleting it (``<userhome>/.conan2``).
 
 
+Install with pipx
+-----------------
+
+In certain scenarios, attempting to install with `pip` may yield the following error:
+
+.. code-block:: bash
+
+    error: externally-managed-environment
+
+    × This environment is externally managed
+    ╰─> To install Python packages system-wide, try apt install
+        python3-xyz, where xyz is the package you are trying to
+        install.
+    ...
+
+
+This is because some modern Linux distributions have started marking their Python
+installations as "externally managed", which means that the system's package manager is
+responsible for managing Python packages. Installing packages globally or even in the user
+space can interfere with system operations and potentially break system tools (check
+[PEP-668](https://peps.python.org/pep-0668/) for more detailed information).
+
+For those cases it's recommended to use `pipx` to install Conan. `pipx` creates a virtual
+environment for each Python application, ensuring that dependencies do not conflict. The
+advantage is that it isolates Conan and its dependencies from the system Python and avoids
+potential conflicts with system packages while providing a clean environment for Conan to
+run.
+
+To install Conan with `pipx`:
+
+1. Ensure `pipx` is installed on your system. If it isn't, check the installation guidelines
```

Comment:
It may be worth adding instructions as an example for an Ubuntu/Debian distro?

the link from pipx doesn't really include instructions specific to Linux, other than `pip install --user pipx` - which really will have similar problems as trying to install Conan itself (in the sense that the environment is externally managed, although maybe it is more relax for `--user`).

on Debian based distros, one can do:
```
apt-get install pipx
pipx ensurepath
```
then restart the shell and install conan via `pipx install conan`

This is probably the safest, and equivalent to doing `brew install pipx`. 

So I would perhaps go beyond that the pipx documentation says, and recommend installing `pipx` via the system package manager, and then use _that_ pipx. Unfortunately it may be the case that the `pipx` package has different names depending on the distro (e.g. `pipx` vs `python3-pipx` or `python-pipx`)